### PR TITLE
add fake block gas ut

### PR DESCRIPTION
**Pull request by zjg555543**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: app/test_helpers.go [hunk 1]
```diff
@@ -32,3 +32,28 @@ func Setup(isCheckTx bool) *OKExChainApp {
 
 	return app
 }
+
+func SetupWithChainId(isCheckTx bool, chainId string) *OKExChainApp {
```
Comment (MinZhu09) [inline app/test_helpers.go]: can use optional parameters in Setup function

Comment (MinZhu09) [inline app/test_helpers.go]: can use optional parameters in Setup function

File: app/app_test.go [hunk 2]
```diff
@@ -68,3 +115,225 @@ func TestProposalManager(t *testing.T) {
...
+				newProposalMsg := gov.NewMsgSubmitProposal(content, sysCoins10, suite.stdSenderAccAddress)
+				depositMsg := gov.NewMsgDeposit(suite.stdSenderAccAddress, govProposalID1, sysCoins90)
+				msgs := []cosmossdk.Msg{newProposalMsg, depositMsg}
+				tx := newTestStdTx(msgs, []crypto.PrivKey{suite.stdSenderPrivKey}, []uint64{accountNum}, []uint64{nonce1}, txFees, memo)
+
+				txEncoder := authclient.GetTxEncoder(suite.codec)
+				txBytes, _ := txEncoder(tx)
+				return txBytes
+			},
+			68007,
```
Comment (MinZhu09) [inline app/app_test.go]: add comment for error code

Comment (MinZhu09) [inline app/app_test.go]: add comment for error code

**Review by giskook — APPROVED** [review]

**Review by ItsFunny — APPROVED** [review]

**Review by MinZhu09 — APPROVED** [review]

**Pull request merged**